src/sentry/static/sentry/app/views/settings/organizationGeneralSettings/organizationSettingsForm.jsx

```diff
@@ -42,14 +41,6 @@ const OrganizationSettingsForm = createReactClass({
             onSave(initialData, model.initialData);
           }
         }}
-        onSubmitError={error => {
```

Comment:
Are there no other failure conditions anymore? Should you keep the `Unable to save change` error?